mani_skill/assets/robots/xarm6/xarm6_nogripper.urdf

```diff
@@ -0,0 +1,250 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
all the URDFs should have some .srdf files which let you define custom collisions between links to ignore. this is likely why the sim speed isn't as fast as it could be as well. Definitely should ignore a lot of these that show up in the viewer if you open the contacts tab, some of them never occur but take up processing time. We by default already ignore adjacent links.

![image](https://github.com/user-attachments/assets/3ee63b8f-cbed-465c-927d-80f4bd7ac238)

See the fetch.srdf file in this repo for an example.

Reply:
Fixed! Added SRDFs for all xarm6 variants. I have added more details about these in the comment [below](https://github.com/haosulab/ManiSkill/pull/636#issuecomment-2434495095).